### PR TITLE
Fixes #729. Non-logged in searches from home page should have all params in URL bar

### LINKED ISSUE #729: All issues page params should be updated, when reached from homepage search
Right now search works, but the model params aren't merged with `q` for some reason.

### PR DESCRIPTION
**Pull request by miketaylr**
:baby: 

r? @magsout

### CODE REVIEW

File: tests/functional/search-auth.js [hunk 1]
```diff
@@ -90,7 +91,29 @@ define([
...
-    }
+    },
 
+    'Search from the homepage': function() {
+      return this.remote
+        .setFindTimeout(intern.config.wc.pageLoadTimeout)
+        .get(require.toUrl(url('/')))
+        .findByCssSelector('.js-SearchBarOpen').click()
+        .end()
+        .findByCssSelector('.js-SearchBar input').click()
```
Comment (magsout) [inline tests/functional/search-auth.js]: it's necessary, autofocus doesn't make the job ?

**Comment by magsout** [comment]
rock_star

File: tests/functional/search-auth.js (hunk 1 above)
Comment (miketaylr) [inline tests/functional/search-auth.js]: Oh yeah, probably not. Forgot!

Comment (miketaylr) [inline tests/functional/search-auth.js]: Hmm, seems like Selenium does it too quickly. I guess we just leave it in. :fried_shrimp:

**Pull request merged**